### ISSUE TITLE
Load Variable does not work properly for integers

### ISSUE THREAD

**Issue by CG-Tespy** [snozbot/fungus#867]
**Describe the bug**
When you try to load an integer variable, and the key doesn't exist, the variable gets set to 0 instead of being unmodified.

**To Reproduce**
1. Create a new scene
2. Create a flowchart in there with an integer variable set to some number other than 0
3. Have a Game Started block that tries to load the integer variable with some key that wasn't registered
4. See as the variable gets set to 0

**Expected behavior**
The variable to be unmodified, as the CommandInfo says.

**Versions & Platform**
 - Unity version: 2019.4.9f1
 - Fungus Version: 3.15
 - Editor OS: Windows 10
 - Build Platform: Desktop Standalone

**Additional context**
I could fix this myself and submit a PR, but rebasing the develop branch might undo the work I've done on another PR I have pending. I suggest changing OnEnter like so:

```
if (key == "" || // Old code. Remove this
            variable == null)
{
    Continue();
    return;
}

// The following should fix the problem
var flowchart = GetFlowchart(); // This chunk was moved from a bit later in OnEnter.
// Prepend the current save profile (if any)
string prefsKey = SetSaveProfile.SaveProfile + "_" + flowchart.SubstituteVariables(key);
if (key == "" || variable == null || !PlayerPrefs.HasKey(prefsKey)) 
{
    Continue();
    return;
}
```

**Comment by stevehalliwell** [comment]
@CG-Tespy You can have multiple PRs pending by making more branches in your fork. Make a new branch from upstream/develop and then once done, the PR is from that branch into develop here. This is the approach I used before I was maintainer and is akin to the approached used in gitflow.

**Comment by CG-Tespy** [comment]
@stevehalliwell Understood, thank you :) I'll have a PR for this particular issue submitted soon

**Comment by CG-Tespy** [comment]
Unfortunately, git doesn't let me do that... When I try to swithc to the upstream/develop branch, it tells me that the branch already exists, before forcing em to switch to the branch I was on previously. This keeps me from making a new branch as you suggested. 

Do you have any ideas as to how to get around this? @stevehalliwell

**Comment by s-halliwell-sae** [comment]
@CG-Tespy not sure which git gui you are using or just doing command line. But this should have what you are after, https://stackoverflow.com/questions/8483983/how-to-create-the-branch-from-specific-commit-in-different-branch/8491176

The error you are describing is trying to tell you that the name of the new branch you are trying to create is already taken.

**Comment by stevehalliwell** [comment]
@CG-Tespy not sure which git gui you are using or just doing command line. But this should have what you are after, https://stackoverflow.com/questions/8483983/how-to-create-the-branch-from-specific-commit-in-different-branch/8491176

The error you are describing is trying to tell you that the name of the new branch you are trying to create is already taken.

(Note, replaced comment as I was logged into the wrong account)
